### DISCUSSION TITLE
Is it possible to silent direnv?

### DISCUSSION THREAD

**Discussion by Demianeen** [direnv/direnv#1277] [Q&A]
I use starship that shows if I use venv or not, and this is my the only usage of direnv. So I don't really see the need to have it shouting that it is loaded or unloaded. Is it possible to silent direnv?

**Comment by ShreeChaturvedi** [comment]
Set `DIRENV_LOG_FORMAT` to empty in your shell rc:

```sh
export DIRENV_LOG_FORMAT=
```

Open a new terminal and `cd` into a project. The load and unload lines should stop.

On 2.36 or newer, also create `~/.config/direnv/direnv.toml` if it isn't there yet, even empty. Without that file, 2.36 ignores the env var.
